cura/CuraActions.py

```diff
@@ -23,5 +35,56 @@ def openBugReportPage(self):
         event = CallFunctionEvent(self._openUrl, [QUrl("http://github.com/Ultimaker/Cura/issues")], {})
         Application.getInstance().functionEvent(event)
 
+    ##  Center all objects in the selection
+    @pyqtSlot()
+    def centerSelection(self) -> None:
+        operation = GroupedOperation()
+        for node in Selection.getAllSelectedObjects():
+            current_node = node
+            while current_node.getParent() and current_node.getParent().callDecoration("isGroup"):
+                current_node = current_node.getParent()
+
+            center_operation = SetTransformOperation(current_node, Vector())
+            operation.addOperation(center_operation)
+        operation.push()
+
+    ##  Multiply all objects in the selection
+    #
+    #   \param count The number of times to multiply the selection.
+    @pyqtSlot(int)
+    def multiplySelection(self, count: int) -> None:
+        job = MultiplyObjectsJob(Selection.getAllSelectedObjects(), count, 8)
+        job.start()
+        return
+
+    ##  Delete all selected objects.
+    @pyqtSlot()
+    def deleteSelection(self) -> None:
+        if not Application.getInstance().getController().getToolsEnabled():
+            return
+
+        removed_group_nodes = []
+        op = GroupedOperation()
+        nodes = Selection.getAllSelectedObjects()
+        for node in nodes:
+            op.addOperation(RemoveSceneNodeOperation(node))
+            group_node = node.getParent()
+            if group_node and group_node.callDecoration("isGroup") and group_node not in removed_group_nodes:
+                remaining_nodes_in_group = list(set(group_node.getChildren()) - set(nodes))
+                if len(remaining_nodes_in_group) == 1:
+                    removed_group_nodes.append(group_node)
+                    op.addOperation(SetParentOperation(remaining_nodes_in_group[0], group_node.getParent()))
+                    op.addOperation(RemoveSceneNodeOperation(group_node))
+        op.push()
+
+    @pyqtSlot(str)
+    def setSelectionExtruder(self, extruder_id: str) -> None:
```

Comment:
The naming is different compared to the others; deleteSelection vs setSelectionExtruder. I'd say it should be osmething like "setExtruderForSelection"  or something like that.